dotnet-desktop-guide/net/winforms/controls-design/extend-existing.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Customize an existing control
+description: Learn how to inherit from existing controls so that another control has all of its functionality and visual properties.
+ms.date: 04/24/2023
+dev_langs:
+  - "csharp"
+  - "vb"
+helpviewer_keywords:
+  - "inheritance [Windows Forms], Windows Forms custom controls"
+  - "custom controls [Windows Forms], inheritance"
+---
+# Extend an existing control
+
+If you want to add more features to an existing control, you can create a control that inherits from an existing control. The new control contains all of the capabilities and visual aspect of the base control, but gives you opportunity to extend it. For example, if you created a control that inherits <xref:System.Windows.Forms.Button>, your new control would look and act exactly like a button. You could create new methods and properties to customize the behavior of the control. Some controls allow you to override the <xref:System.Windows.Forms.Control.OnPaint%2A> method to change the way the control looks.
+
+## Add a custom control to a project
+
+After creating a new project, use the Visual Studio templates to create a user control. The following steps demonstrate how to add a user control to your project:
+
+01. In Visual Studio, find the **Project Explorer** pane. Right-click on the project and choose **Add** > **User Control (Windows Forms)**.
```

Comment:
Am I missing something? The steps are for adding a UserControl but this is supposed to be extending an existing control. The image is the correct one but the text in step 1 is user control.

Also in the steps I didn't notice actually extending the button class.

Reply:
OK I modified the wording here to link this section to the next.